### ISSUE TITLE
Add support for binding to specified address and port

### ISSUE THREAD

**Issue by MrDefacto** [tykling/gstat_exporter#6]
```
--- gstat_exporter.py	2023-07-15 16:24:37.216155000 +0200
+++ gstat_exporter.py	2023-07-15 16:21:36.671814000 +0200
@@ -1,4 +1,5 @@
+import argparse
 from prometheus_client import start_http_server, Gauge  # type: ignore
 from subprocess import Popen, PIPE
 from typing import Dict
@@ -407,6 +408,19 @@
     ],
 )

-start_http_server(9248)
-while True:
-    process_request()
+if __name__ == "__main__":
+    # Initialize parser
+    parser = argparse.ArgumentParser(description="Server address and port.")
+
+    # Add parameters
+    parser.add_argument("-a", "--address", help="Address to run the server on.", default="127.0.0.1")
+    parser.add_argument("-p", "--port", help="Port to run the server on.", default=9248, type=int)
+
+    # Parse arguments
+    args = parser.parse_args()
+
+    # Use the provided address and port
+    start_http_server(args.port, args.address)
+
+    while True:
+        process_request()
```

**Comment by tykling** [comment]
thank you, I added something like this code in 5e9b5699a39cf44bd5dd7d8f7d07a9942cf668cf

**Comment by tykling** [comment]
this is available in the v0.2.0 release tagged today https://github.com/tykling/gstat_exporter/releases/tag/v0.2.0
